betelgeuse.py

```diff
@@ -248,7 +248,12 @@ def add_test_case(args):
         if test.docstring:
             if not type(test.docstring) == unicode:
                 test.docstring = test.docstring.decode('utf8')
-            test.docstring = RST_PARSER.parse(test.docstring)
+            test.docstring = RST_PARSER.parse(str(test.docstring))
```

Comment:
I fixed this as `make test` failed with error 

    TypeError: coercing to Unicode: need string or buffer